README.md

```diff
@@ -1,3 +1,4 @@
-Welcome to the KILT workshop!  
 
-### 👉 Workshop: https://kiltprotocol.github.io/kilt-workshop-101/#/ 
+Welcome to the KILT tutorial!  
+
+### 👉 Tutorial: https://kiltprotocol.github.io/kilt-workshop-101/#/ 
```

Comment:
You are relinking the tutorial. So you go back into it. 

Would we need a link to itself or maybe a link to something else here? Such as the source code?

Reply:
Good catch - I realized that the README page, that I've updated for other reasons, is also automatically used as the first page of the tutorial. This is fixed 👍